### ISSUE TITLE
Tower SSH Error to Linux nodes

### ISSUE THREAD

**Issue by wintechdevops** [ansible/ansible#13916]
I have got Ansible and Ansible Tower to make request to Windows nodes, but I can only run jobs to Linux machines if I use Ansible. If I create a job in Ansible Tower to a Linux machine I am getting the following error:

Using /etc/ansible/ansible.cfg as config file
SSH password: 
<10.10.3.166> ESTABLISH SSH CONNECTION FOR USER: root
<10.10.3.166> SSH: EXEC sshpass -d15 ssh -C -q -o StrictHostKeyChecking=no -o User=root -o ConnectTimeout=10 -tt 10.10.3.166 'mkdir -p "$( echo $HOME/.ansible/tmp/ansible-tmp-1452875343.81-272367721346512 )" && echo "$( echo $HOME/.ansible/tmp/ansible-tmp-1452875343.81-272367721346512 )"'
<10.10.3.166> PUT /tmp/tmphjOKTN TO /root/.ansible/tmp/ansible-tmp-1452875343.81-272367721346512/command
<10.10.3.166> SSH: EXEC sshpass -d15 sftp -b - -C -o StrictHostKeyChecking=no -o User=root -o ConnectTimeout=10 '[10.10.3.166]'
10.10.3.166 | UNREACHABLE! => { "changed": false, "msg": "ERROR! SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh", "unreachable": true }

Ansible and Ansible Tower are both installed on the same CentOS machine.
Running the latest versions of both Ansible and Ansible Tower.

Please can you tell me what could be causing this?

**Comment by bcoca** [comment]
for tower support go to http://support.ansible.com or email support@ansible.com

**Comment by wintechdevops** [comment]
I did email support? Not sure why this ticket was closed?

When I run a ping command from the ansible server everything works. This is a ping command to a CentOS machine.
![capture](https://cloud.githubusercontent.com/assets/16669668/12388779/e5f29200-bdcc-11e5-927a-40185ad712ae.JPG)

But when I try and run a simple command to that same CentOS machine I get a connection error.
![capture1](https://cloud.githubusercontent.com/assets/16669668/12388801/0174e58c-bdcd-11e5-9617-81f79beb2237.JPG)

Tower version:   2.4.3
Ansible Version: 2.0.0.1
CentOS Version: 7
